var/spack/repos/builtin/packages/libcroco/package.py

```diff
@@ -15,16 +15,18 @@ class Libcroco(AutotoolsPackage):
     version("0.6.13", sha256="767ec234ae7aa684695b3a735548224888132e063f92db585759b422570621d4")
     version("0.6.12", sha256="ddc4b5546c9fb4280a5017e2707fbd4839034ed1aba5b7d4372212f34f84f860")
 
-    variant("doc", default=False, description="Build documentation with gtk-doc")
+    # libcroco has a --enable-gtk-doc configure flag that appears to be
+    # ignored as of version 0.6.13. Until that flag is honored, the +doc
+    # variant is a no-op
+    # variant("doc", default=False,
```

Comment:
Is it worth making this a conditional variant that only applies to prior versions?

https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-variants

Reply:
I think it applies t all variants at this point. I tried to contact the libcroco developers to fix this, but they haven't responded. From what I can see in their public repo, this still isn't fixed. Perhaps if/when it is fixed, then we can make it a conditional variant on spack.

Reply:
@tldahlgren do you still think we should make this a conditional variant?

Reply:
No.

Reply:
Thanks for the "ping".